test/DynamoCoreWpfTests/GraphLayoutTests.cs

```diff
@@ -32,13 +32,7 @@ public void GraphLayoutEmpty()
 
             ViewModel.DoGraphAutoLayout(null);
 
-            Assert.IsNull(ViewModel.CurrentSpace.LayoutGraph);
```

Comment:
Does this implementation change the way the overall graph is laid out? What I mean is why do the existing tests need to be updated since they maybe pertaining to graph layout of overall graph?